apps/users/forms/profile.py

```diff
@@ -153,16 +158,6 @@ class Meta:
             "newsletter_recipient",
         ]
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.fields["nationality"].choices = [("", "Please select your nationality")] + list(
-            self.fields["nationality"].choices
-        )[1:]
-        self.fields["department"].choices = [("", "Please select your department")] + list(
-            self.fields["department"].choices
-        )[1:]
-        self.fields["program"].choices = [("", "Please select your program")] + list(self.fields["program"].choices)[1:]
```

Comment:
Do we want to add an empty choice? Or can we do that in the HTML?

Reply:
this is the update form, so it shouldnt matter as the default value is the value given at registration. The empty choice is not a valid entry, just the first option presented during registration.